docs/product-assembly_pam-on-openshift/src/main/asciidoc/main.adoc

```diff
@@ -0,0 +1,30 @@
+[id='assembly_dm-7.0-release-notes']
+
+include::product-shared-docs/document-attributes.adoc[]
+
+= Deploying {PRODUCT} {PRODUCT_VERSION} on OpenShift
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:imagesdir: topics/product-shared-docs/images
+:context: dm-7.0-release-notes
+include::product-shared-docs/author-group.adoc[]
+
+// Purpose statement for the assembly
+These release notes highlight new features in {PRODUCT} {PRODUCT_VERSION}, list features that are in technical preview, and include any known issues.
+
+include::ba-openshift-overview-con.adoc[leveloffset=+1]
+include::product-assembly_dm-on-openshift/dm-openshift-prepare-con.adoc[leveloffset=+1]
```

Comment:
I'm not seeing this file, but based on the rendered version, I wonder if the following, presumably from this file, is missing proper parametrization:

decision services or for other decision services